translations/binaries/slps_strings_items_consumable.json

```
@@ -0,0 +1,2153 @@
+            "offset": 1761000,
+            "string": "「�」の文字が見える不思議な球体。[LINE]\n全ての「アーマー」を知りし者のみ[LINE]\n答えを知る。[END]",
+            "translation": "「�」의 문자가 보이는 이상한 구체.[LINE]\n모든「갑옷」을 알고 있는 자만[LINE]\n답을 알 수 있다.[END]",
+            "original_bytes_length": 81,
+            "available_bytes_size": 87
+        },
+        {
+            "offset": 1760976,
+            "string": "[Sign_Box]エプタスフィア[END]",
+            "translation": "[Sign_Box]에프타스피아[END]",
+            "original_bytes_length": 20,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1760888,
+            "string": "「�」の文字が見える不思議な球体。[LINE]\n全ての「プレート」を知りし者のみ[LINE]\n答えを知る。[END]",
+            "translation": "「�」의 문자가 보이는 이상한 구체.[LINE]\n모든「판금」을 알고 있는 자만[LINE]\n답을 알 수 있다.[END]",
+            "original_bytes_length": 81,
+            "available_bytes_size": 87
+        },
+        {
+            "offset": 1760864,
+            "string": "[Sign_Box]オクトスフィア[END]",
+            "translation": "[Sign_Box]옥토스피아[END]",
+            "original_bytes_length": 20,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1760784,
+            "string": "「�」の文字が見える不思議な球体。[LINE]\n全ての「ウェア」を知りし者のみ[LINE]\n答えを知る。[END]",
+            "translation": "「�」의 문자가 보이는 이상한 구체.[LINE]\n모든「의상」을 알고 있는 자만[LINE]\n답을 알 수 있다.[END]",
+            "original_bytes_length": 79,
+            "available_bytes_size": 79
+        },
+        {
+            "offset": 1760760,
+            "string": "[Sign_Box]エンネアスフィア[END]",
+            "translation": "[Sign_Box]엔네아스피아[END]",
+            "original_bytes_length": 22,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1760672,
+            "string": "「�」の文字が見える不思議な球体。[LINE]\n全ての「クローク」を知りし者のみ[LINE]\n答えを知る。[END]",
+            "translation": "「�」의 문자가 보이는 이상한 구체.[LINE]\n모든「망토」를 알고 있는 자만[LINE]\n답을 알 수 있다.[END]",
+            "original_bytes_length": 81,
+            "available_bytes_size": 87
+        },
+        {
+            "offset": 1760648,
+            "string": "[Sign_Box]デカスフィア[END]",
+            "translation": "[Sign_Box]데카스피아[END]",
+            "original_bytes_length": 18,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1760568,
+            "string": "「�」の文字が見える不思議な球体。[LINE]\n全ての「ローブ」を知りし者のみ[LINE]\n答えを知る。[END]",
+            "translation": "「�」의 문자가 보이는 이상한 구체.[LINE]\n모든「로브」를 알고 있는 자만[LINE]\n답을 알 수 있다.[END]",
+            "original_bytes_length": 79,
+            "available_bytes_size": 79
+        },
+        {
+            "offset": 1760552,
+            "string": "[Sign_Box]まんぼう[END]",
+            "translation": "[Sign_Box]만봉[END]",
+            "original_bytes_length": 14,
+            "available_bytes_size": 15
+        },
+        {
+            "offset": 1760480,
+            "string": "まんぼう……所持数に応じて[LINE]\nリリスのまんぼう戦吼の威力が[LINE]\nアップ！[END]",
+            "translation": "만봉…… 소지 수에 따라[LINE]\n릴리스의 만봉 전투 함성의 힘이[LINE]\n상승![END]",
```

```suggestion
            "translation": "맘보…… 소지 수에 따라[LINE]\n리리스의 맘보전후의 힘이[LINE]\n상승![END]",
```

리리스의 '맘보전후'. 스탄의 기술 중 '사자전후'의 패러디로 추정